crates/router/src/connector/payone/transformers.rs

```diff
@@ -1,227 +1,298 @@
         match item {
-            RefundStatus::Succeeded => Self::Success,
-            RefundStatus::Failed => Self::Failure,
-            RefundStatus::Processing => Self::Pending,
-            //TODO: Review mapping
+            PayoutMethodData::Card(card) => Ok(Self(
+                Card {
+                    card_number: card.card_number.clone(),
+                    card_holder_name: card
+                        .card_holder_name
+                        .clone()
+                        .get_required_value("card_holder_name")
+                        .change_context(errors::ConnectorError::MissingRequiredField {
+                            field_name: "payout_method_data.card.holder_name",
+                        })?,
+                    expiry_date: match card.get_expiry_date_as_mmyy() {
+                        Ok(date) => date,
+                        Err(_) => Err(errors::ConnectorError::MissingRequiredField {
+                            field_name: "payout_method_data.card.expiry_date",
+                        })?,
+                    },
+                },
+                card.get_card_issuer()?,
+            )),
+            _ => Err(errors::ConnectorError::MissingRequiredField {
+                field_name: "payout_method_data.card",
+            })?,
         }
     }
 }
 
-//TODO: Fill the struct with respective fields
-#[derive(Default, Debug, Clone, Serialize, Deserialize)]
-pub struct RefundResponse {
+#[cfg(feature = "payouts")]
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum PayoneStatus {
+    Created,
+    #[default]
+    PendingApproval,
+    Rejected,
+    PayoutRequested,
+    AccountCredited,
+    RejectedCredit,
+    Cancelled,
+    Reversed,
+}
+#[cfg(feature = "payouts")]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PayoneTransferDetails {
+    transfer_purpose: Option<String>,
+    source_of_funds: Option<String>,
+    transfer_purpose_sub_transfer_purpose: Option<String>,
+}
+
+#[allow(dead_code)]
+#[cfg(feature = "payouts")]
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PayonePayoutFulfillResponse {
     id: String,
-    status: RefundStatus,
+    payout_output: PayoutOutput,
+    status: PayoneStatus,
 }
 
-impl TryFrom<types::RefundsResponseRouterData<api::Execute, RefundResponse>>
-    for types::RefundsRouterData<api::Execute>
-{
-    type Error = error_stack::Report<errors::ConnectorError>;
-    fn try_from(
-        item: types::RefundsResponseRouterData<api::Execute, RefundResponse>,
-    ) -> Result<Self, Self::Error> {
-        Ok(Self {
-            response: Ok(types::RefundsResponseData {
-                connector_refund_id: item.response.id.to_string(),
-                refund_status: enums::RefundStatus::from(item.response.status),
-            }),
-            ..item.data
-        })
+#[allow(dead_code)]
```

Comment:
Why `dead_code`?

Reply:
Also `Serialize` is not required in the derives as this is response.
Please take care of this at all the applicable places